src/app/[locale]/(components)/localeMenu.tsx

```diff
@@ -21,6 +21,7 @@ export const LocaleMenu: FC = () => {
 
   return (
     <Select
+      aria-label="language selector"
```

Comment:
Created #98 as a follow up to this PR to translate these (not required to merge)